ldaptor/test/test_ldiftree.py

```diff
@@ -751,6 +762,22 @@ def cb3(got):
         d.addCallback(cb3)
         return d
 
+    @_fromCoroutineFunction
+    async def test_diffTree_edit_failure(self):
+        otherDir = self.mktemp()
```

Comment:
This needs a docstring to explain what is tested here and what  is the scope/purpose of this test.

Reply:
most tests don't have a docstring, and it's pretty clear it's testing an edit failure

Reply:
ok... but which edit failure.. is a general edit failure or a specific one.